app/components/cards/chat_online_component.rb

```diff
@@ -16,15 +16,10 @@ def header
 
     def snippet
       @snippet ||= <<~SNIPPET
-        If you have questions about getting into teaching, we can help you get
-        the answers
+        Chat online with one of our teacher training experts. They're available Monday-Friday between 8:30am and 5:30pm.
```

Comment:
```suggestion
        Chat online with one of our teacher training experts. They're available Monday to Friday between 8:30am and 5:30pm.
```

The GOV.UK design system recommends using 'to' rather than a dash as it is quicker to read and better for screen readers.